study.md

```diff
@@ -134,15 +134,15 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a brief description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+HTTP is the set of rules (protocols) that determine how web pages are displayed and formatted
 ```
 
 ## Describe what a client is and what a server is
 
  What is a client and what is a server? How do they interact with each other?
 
 ```md
-<!-- your answer here -->
+The client is basically your personal device, say a laptop, and it sends a request to the server and gets a response from a server. A server is just a computer that stores things.
```

Comment:
Good but the client can actually be the browser (Chrome, Safari) not just the machine it is running on.